fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystem
+metadata:
+  name: rsphome
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+    quotas:
+      maxSize: 10Gi
+  dataPools:
+    - name: default
+      failureDomain: host
+      replicated:
+        size: 3
+      quotas:
+        maxSize: 25Gi
+    - name: ec
+      failureDomain: host
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 3
```

Comment:
As pillan is a 9 node cluster, 6+3 is probably excess as a single node going down would prevent writes.   K + M <= nodes - 1

Reply:
but i think this cannot be changed now, right?

Reply:
No because 6+3 = 9

Reply:
this was changed 👍🏻